qa/tasks/cephfs/fuse_mount.py

```diff
@@ -89,7 +89,8 @@ def _run_mount_cmd(self, mntopts, check_status):
             stdin=self._mount_cmd_stdin,
             stdout=mountcmd_stdout,
             stderr=mountcmd_stderr,
-            wait=False
+            wait=False,
+            omit_sudo=False
```

Comment:
This fixes the  `mount: /sys/fs/fuse/connections: must be superuser to use mount.` error. 

But there is a new error:

```
2022-08-10 17:26:35,637.637 INFO:tasks.cephfs.fuse_mount:Found client admin socket at /home/jcollin/workspace/ceph/build/asok/client.0.143449.asok
2022-08-10 17:26:35,637.637 DEBUG:__main__:"sudo" was omitted from the following cmd args before execution and logging using function overriding; check vstart_runner.py for more details.
2022-08-10 17:26:35,637.637 DEBUG:__main__:> sudo ./bin/ceph --admin-daemon /home/jcollin/workspace/ceph/build/asok/client.0.143449.asok status
*** DEVELOPER MODE: setting PATH, PYTHONPATH and LD_LIBRARY_PATH ***
admin_socket: exception getting command descriptions: [Errno 13] Permission denied
2022-08-10 17:26:37,737.737 DEBUG:__main__:"sudo" was omitted from the following cmd args before execution and logging using function overriding; check vstart_runner.py for more details.
2022-08-10 17:26:37,737.737 DEBUG:__main__:> sudo ./bin/ceph --admin-daemon /home/jcollin/workspace/ceph/build/asok/client.0.143449.asok status
*** DEVELOPER MODE: setting PATH, PYTHONPATH and LD_LIBRARY_PATH ***
admin_socket: exception getting command descriptions: [Errno 13] Permission denied
```

So I've removed `omit_sudo=False` and the test passed.

Reply:
Strange. I did not hit this issue. What test were you running?

Reply:
I ran the following tests, both error out.

```
 2465  10/08/22 17:24:18 python ../qa/tasks/vstart_runner.py tasks.cephfs.test_mds_metrics.TestMDSMetrics.test_perf_stats_stale_metrics --debug
```
```
 2466  10/08/22 17:26:13  python ../qa/tasks/vstart_runner.py tasks.cephfs.test_mds_metrics.TestMDSMetrics.test_metrics_from_rank --debug
```

Reply:
Runs fine for me -

```
2022-08-11 15:15:22,692.692 INFO:__main__:Stopped test: test_perf_stats_stale_metrics (tasks.cephfs.test_mds_metrics.TestMDSMetrics)
That `ceph fs perf stats` doesn't output stale metrics after the rank0 MDS failover in 66.651653s
2022-08-11 15:15:22,692.692 INFO:__main__:test_perf_stats_stale_metrics (tasks.cephfs.test_mds_metrics.TestMDSMetrics)
2022-08-11 15:15:22,692.692 INFO:__main__:That `ceph fs perf stats` doesn't output stale metrics after the rank0 MDS failover ... ok
2022-08-11 15:15:22,692.692 INFO:__main__:
2022-08-11 15:15:22,693.693 INFO:__main__:----------------------------------------------------------------------
2022-08-11 15:15:22,693.693 INFO:__main__:Ran 1 test in 66.652s
```

Reply:
Inspecting log messages you copied here, seems like ceph-fuse command was executed with superuser privileges which shouldn't be the case because `sudo` will removed by this patch. See https://github.com/ceph/ceph/pull/47533/files#r943310375.

Maybe some commits of this PR were missing it was tested locally?